### PR TITLE
fix RCTPerfMonitor display unit error

### PR DESCRIPTION
**Pull request by ywz2010**
The  RCTPerformanceLogger log the time in ms not us. Especially the unit of RCTPLBundleSize is byte.

### CODE REVIEW

**Comment by nicklockwood** [comment]
cc @tadeuzagallo

File: React/Profiler/RCTPerfMonitor.m [hunk 1]
```diff
@@ -505,8 +505,13 @@ - (void)loadPerformanceLoggerData
   NSArray *times = RCTPerformanceLoggerOutput();
   NSUInteger i = 0;
   for (NSString *label in RCTPerformanceLoggerLabels()) {
-    [data addObject:[NSString stringWithFormat:@"%@: %lldus", label,
-                     [times[i+1] longLongValue] - [times[i] longLongValue]]];
+    if([label isEqualToString:@"BundleSize"]){
+      [data addObject:[NSString stringWithFormat:@"%@: %lldByte", label,
```
Comment (tadeuzagallo) [inline React/Profiler/RCTPerfMonitor.m]: nit: I think the sufix could simply be `b`.

Comment (ywz2010) [inline React/Profiler/RCTPerfMonitor.m]: `b` is better, so change the `Byte` to `b`

**Comment by nicklockwood** [comment]
@facebook-github-bot shipit

**Pull request closed without merging**